articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: engineering-education
+status: review
+published: false
+url: /engineering-education/getting-started-with-browser-sync/
+title: Getting Started with Browser-sync
+description: The basic concept of browser-sync, installing browser-sync and using it serve your files while building a website project
+author: aransiola-ayodele-leom
+date: 2020-09-24T00:00:27-21
+topics: [Browser-sync]
+excerpt_separator: <!--more-->
+images:
+ 
+ - url: /engineering-education/getting-started-with-browser-sync/hero.jpg
+   alt: Computers in a workspace displaying code
+---
+ 
+# Getting Started with Browser-sync
+ 
+ 
+Browser-sync is an automation tool that allows you to keep multiple browsers & devices in sync when building websites. It also gives your testing workflow faster synchronization, interaction, and code change across multiple devices easily, both internally and externally.
+ 
+>I have been working with different local servers just to replace the hassle of the live server. After trying out many, I found [browser-sync](https://www.browsersync.io/) more fascinating because of its complete packages. A stitch in time saves nine, which is Browser-sync in the quote.
+ 
+>Browser-sync is awesomely fast, with incredible features for your use. Browser-sync allows you to also check your work preview simultaneously on different devices, provided you are connected on the same network.
+ 
+![Internal-UI-Interface](/engineering-education/getting-started-with-browser-sync/browsersync-dashboard.png)
+ 
+In the above image, you will see different features, these are the awesome things browser-sync provides you. Install on your computer and enjoy a seamless watch over your files. It also records your test URLs so you can push them back out to all devices with a single click.
+I will give the steps to install browser-sync for Linux. Other OS installation can be seen in their documentation [here](https://www.browsersync.io/docs)
+ 
+### Linux Installation
+ 
+The first step is to install Node.js on your computer
+```
+sudo apt install nodejs
+ 
+```
+After the installation, check the version of the installed node using the command below
+ 
+```
+node -v
+```
+ 
+Once you have successfully installed nodejs on your computer, let us get you started on how to install browser-sync.
+ 
+```
+npm install -g browser-sync
+```
+The above command will get browser-sync installed on your computer. After that, change the directory to where your project file is located. Looking at the image below, the project file is located on my Desktop. So, I do a “cd Desktop/Vue project” in my terminal. This takes me to the Desktop directory of my computer and inside the Vue project folder.
+ 
+![Change-directory](/engineering-education/getting-started-with-browser-sync/cd.png)
+ 
+The next thing to do is run the command below:
+ 
+```
+browser-sync start --server --directory --files "*"
```

Comment:
For code blocks like this, add the language afterwards to activate syntax highlighting. For console commands, for example : ```bash

Please update similar commands.